src/OpenLoco/Company.h

```diff
@@ -97,11 +97,13 @@ namespace OpenLoco
         currency48_t vehicleProfit;                  // 0x8B9E
         uint16_t transportTypeCount[6];              // 0x8BA4
         uint8_t var_8BB0[9];
-        uint8_t pad_8BB9[0x8BBC - 0x8BB9];
+        uint8_t observation_status;   // 0x8BB9;
```

Comment:
Can you introduce an enum class for this.
Also make sure all new fields are lower camelCase.